src/main/java/io/confluent/kafka/connect/datagen/DatagenConnectorConfig.java

```diff
@@ -17,13 +17,23 @@
 package io.confluent.kafka.connect.datagen;
 
 import com.google.common.collect.ImmutableList;
+import io.confluent.kafka.connect.datagen.DatagenTask.Quickstart;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
 import java.util.List;
 import java.util.Map;
-
+import org.apache.avro.Schema;
+import org.apache.avro.Schema.Parser;
+import org.apache.avro.SchemaParseException;
 import org.apache.kafka.common.config.AbstractConfig;
 import org.apache.kafka.common.config.ConfigDef;
-import org.apache.kafka.common.config.ConfigDef.Type;
 import org.apache.kafka.common.config.ConfigDef.Importance;
+import org.apache.kafka.common.config.ConfigDef.Type;
+import org.apache.kafka.common.config.ConfigDef.Validator;
+import org.apache.kafka.common.config.ConfigException;
+import org.apache.kafka.connect.errors.ConnectException;
 
 public class DatagenConnectorConfig extends AbstractConfig {
```

Comment:
Yeah, these methods were intentionally made static since they're generic helper methods. We can't make these instance methods due to the issue outlined above.